packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -1186,6 +1191,16 @@
                     "required": false
                 }
             ]
+        },
+        {
+            "name": "amazonq_stopCodeGeneration",
+            "description": "User stopped the code generation",
+            "metadata": [
+                {
+                    "type": "tabID",
```

Comment:
What information will having a tabID give us? Isn't it just a random string like xcfg2? Instead does it make sense to use the telemetry.ui_click metric instead?

Reply:
I'm pulling @kelvin-klchu to answer these